Automattic/jetpack
# Remove the jQuery Spin and spin.js libraries

Fixes #13555 

#### Changes proposed in this Pull Request:

* Remove the `spin.js` and `jquery.spin` libraries in favor of CSS animations.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Removes a legacy library.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






_Carousel_

* Go to Jetpack Settings > Writing > Display images in a full-screen carousel gallery
* Make sure "Show comments area in carousel" is enabled.
* Create a post with a gallery.
* Navigate to the created post on the front-end.
* Click one of the images.
* At the bottom of the carousel, there is a form for adding a comment.
* Try adding a comment while logged in and logged out.
* A spinner should be shown when you hit "Post Comment".

_Comics_

* Update your theme's `functions.php` and add `add_theme_support( 'jetpack-comic' );`
* In WP Admin, create a new Comics post.
* Navigate to front-end while still logged in.
* Drag and drop an images from your computer to the comic page.
* A drop zone should be highlighted.
* When you drop the image, a spinner should be shown.

#### Proposed changelog entry for your changes:

* Removes the spin.js and jQuery spin libraries in favor of CSS animations.
